_posts/en/newsletters/2021-08-04-newsletter.md

```diff
@@ -0,0 +1,83 @@
+
+*A weekly [series][series preparing for taproot] about how developers
+and service providers can prepare for the upcoming activation of taproot
+at block height {{site.trb}}.*
+
+{% include specials/taproot/en/06-multisignatures.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.10.1rc1][C-Lightning 0.10.1] is a release candidate for
+  an upgrade that contains a number of new features, several bug fixes,
+  and a few updates to developing protocols (including [dual
+  funding][topic dual funding] and [offers][topic offers]).
+
+- [Bitcoin Core 22.0rc2][bitcoin core 22.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software. Major changes in this new
+  version include support for [I2P][topic anonymity networks] connections,
+  removal of support for [version 2 Tor][topic anonymity networks] connections,
+  support for [bech32m address encoding][topic bech32], and enhanced support for
+  hardware wallets.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #22006][] adds [documentation][tracing doc] for User-Space, Statically Defined
+  Tracing (USDT) and the first three tracepoints - build support and macros for which were added in
+[Bitcoin Core #19866][]. Users that build Bitcoin Core with eBPF tracing enabled can hook into the
+tracepoints with the provided [example scripts][contrib tracing doc] or write their own tracing
+scripts for greater observability into the node when a new block is connected, inbound P2P messages
+are received, and outbound P2P messages are sent. The documentation also includes usage examples and
+guidelines for the addition of new tracepoints.
+
+- [Eclair #1893][] Different default relay fees for announced and unannounced channels FIXME:bitschmidty
+
+- [Rust-Lightning #967][] adds support for making [keysend-style spontaneous
+  payments][topic spontaneous payments#add-data-to-the-routing-packet] via the
+  `send_spontaneous_payment` function call. With this change, all four LN
+  implementations we cover will have support for keysend.
+
+  The author has also submitted [corresponding documentation][BOLTs #892] (yet
+  unmerged) on keysend payments as a bLIP (Bitcoin Lightning Improvement
+  Proposals), a new way to document features and best practices which do not
+  belong as part of the spec.
```

Comment:
Should we make it clear that bLIPs are still under discussion and not a sure thing yet?

Reply:
That'd probably be good.  I think it might be sufficient to link to https://bitcoinops.org/en/newsletters/2021/07/07/#blips

Also, I'd prefer we capitalize it as "BLIP" (there's a comment with explanation in the source code of the above newsletter; I'll add it to the style guide if BLIPs do become a thing).